src/ui/FacetRange/FacetRange.ts

```diff
@@ -79,6 +78,23 @@ export class FacetRange extends Facet implements IComponentBindings {
      * > The index cannot automatically generate range values for a `FacetRange` whose [`field`]{@link Facet.options.field} option value references a dynamic field generated by a [query function](https://docs.coveo.com/en/232/). In such a case, you _must_ use the `ranges` option.
      */
     ranges: ComponentOptions.buildJsonOption<IRangeValue[]>(),
+
+    /**
+     * Specifies how to format the the range value.
+     *
+     * The available formats are defined in the Globalize library (see
+     * [Globalize](https://github.com/klaaspieter/jquery-global#globalizeformat-value-format-culture-).
+     *
+     * The most commonly used formats are:
+     * - `c0` - Formats the value as a currency.
+     * - `n0` - Formats the value as an integer.
+     * - `n2` - Formats the value as a floating point with 2 decimal digits.
+     *
+     * **Note:**
+     * Does not work when the [`valueCaption`]{@link Facet.options.valueCaption} option is defined.
+     * Does not work with date type values.
```

Comment:
Could it be clearer to mention the data type where it would work? A white-list instead of a black-list if it's not too long. e.g. `Works only for number values`?